<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>524</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>291</width>
      <height>501</height>
     </rect>
    </property>
    <property name="title">
     <string>pr1266</string>
    </property>
    <widget class="QPushButton" name="backspace">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(255, 204, 0);
color:black;
border:1px solid gray;
}

QPushButton:pressed{
background-color:qlineagradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #FF7832 , stop: 1 #FF9739);
}</string>
     </property>
     <property name="text">
      <string>backspace</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dot">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>430</y>
       <width>70</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(89,89,89);
color:white;
border:1px solid gray;
}
QPushButton::pressed{
background-color:qlineargadient(x1:0,y1:0;x2:0,y2:1,stop: 0 #dadbde , stop 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(255, 204, 0);
color:black;
border:1px solid gray;
}

QPushButton:pressed{
background-color:qlineagradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #FF7832 , stop: 1 #FF9739);
}</string>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number8">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(89,89,89);
color:white;
border:1px solid gray;
}
QPushButton::pressed{
background-color:qlineargadient(x1:0,y1:0;x2:0,y2:1,stop: 0 #dadbde , stop 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number0">
     <property name="geometry">
      <rect>
       <x>79</x>
       <y>430</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(89,89,89);
color:white;
border:1px solid gray;
}
QPushButton::pressed{
background-color:qlineargadient(x1:0,y1:0;x2:0,y2:1,stop: 0 #dadbde , stop 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(89,89,89);
color:white;
border:1px solid gray;
}
QPushButton::pressed{
background-color:qlineargadient(x1:0,y1:0;x2:0,y2:1,stop: 0 #dadbde , stop 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="plus">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>360</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(255, 204, 0);
color:black;
border:1px solid gray;
}

QPushButton:pressed{
background-color:qlineagradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #FF7832 , stop: 1 #FF9739);
}</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(89,89,89);
color:white;
border:1px solid gray;
}
QPushButton::pressed{
background-color:qlineargadient(x1:0,y1:0;x2:0,y2:1,stop: 0 #dadbde , stop 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(89,89,89);
color:white;
border:1px solid gray;
}
QPushButton::pressed{
background-color:qlineargadient(x1:0,y1:0;x2:0,y2:1,stop: 0 #dadbde , stop 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>281</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mj_TV Bold</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLable{
qproperty-alignment:'AlignVCenter | AlignRight';
border:1px solid gray;
}

background-color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="radikal">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(89,89,89);
border:1px solid gray;
color:white;
}
QPushButton::pressed{
background-color:qlineargadient(x1: 0, y1: 0 , x2: 0 , y2: 1, stop : 0 #BEBEBE , stop : 1 #D7D7D7);
}</string>
     </property>
     <property name="text">
      <string>sqrt</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>290</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(89,89,89);
color:white;
border:1px solid gray;
}
QPushButton::pressed{
background-color:qlineargadient(x1:0,y1:0;x2:0,y2:1,stop: 0 #dadbde , stop 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pow">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(89,89,89);
border:1px solid gray;
color:white;
}
QPushButton::pressed{
background-color:qlineargadient(x1: 0, y1: 0 , x2: 0 , y2: 1, stop : 0 #BEBEBE , stop : 1 #D7D7D7);
}</string>
     </property>
     <property name="text">
      <string>2th power</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zarb">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>220</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(255, 204, 0);
color:black;
border:1px solid gray;
}

QPushButton:pressed{
background-color:qlineagradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #FF7832 , stop: 1 #FF9739);
}</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="taghsim">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>150</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(255, 204, 0);
color:black;
border:1px solid gray;
}

QPushButton:pressed{
background-color:qlineagradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #FF7832 , stop: 1 #FF9739);
}</string>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sign">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>70</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(89,89,89);
color:white;
border:1px solid gray;
}
QPushButton::pressed{
background-color:qlineargadient(x1:0,y1:0;x2:0,y2:1,stop: 0 #dadbde , stop 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>+/-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minus">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(255, 204, 0);
color:black;
border:1px solid gray;
}

QPushButton:pressed{
background-color:qlineagradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #FF7832 , stop: 1 #FF9739);
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="equal">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>430</y>
       <width>70</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(255, 204, 0);
color:black;
border:1px solid gray;
}

QPushButton:pressed{
background-color:qlineagradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #FF7832 , stop: 1 #FF9739);
}</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inverse">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(89,89,89);
border:1px solid gray;
color:white;
}
QPushButton::pressed{
background-color:qlineargadient(x1: 0, y1: 0 , x2: 0 , y2: 1, stop : 0 #BEBEBE , stop : 1 #D7D7D7);
}</string>
     </property>
     <property name="text">
      <string>1/x</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>360</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(89,89,89);
color:white;
border:1px solid gray;
}
QPushButton::pressed{
background-color:qlineargadient(x1:0,y1:0;x2:0,y2:1,stop: 0 #dadbde , stop 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>360</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(89,89,89);
color:white;
border:1px solid gray;
}
QPushButton::pressed{
background-color:qlineargadient(x1:0,y1:0;x2:0,y2:1,stop: 0 #dadbde , stop 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>290</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(89,89,89);
color:white;
border:1px solid gray;
}
QPushButton::pressed{
background-color:qlineargadient(x1:0,y1:0;x2:0,y2:1,stop: 0 #dadbde , stop 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="number9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(89,89,89);
color:white;
border:1px solid gray;
}
QPushButton::pressed{
background-color:qlineargadient(x1:0,y1:0;x2:0,y2:1,stop: 0 #dadbde , stop 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ce">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(255, 204, 0);
color:black;
border:1px solid gray;
}

QPushButton:pressed{
background-color:qlineagradient(x1:0,y1:0,x2:0,y2:1, stop: 0 #FF7832 , stop: 1 #FF9739);
}</string>
     </property>
     <property name="text">
      <string>CE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="darsad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(89,89,89);
border:1px solid gray;
color:white;
}
QPushButton::pressed{
background-color:qlineargadient(x1: 0, y1: 0 , x2: 0 , y2: 1, stop : 0 #BEBEBE , stop : 1 #D7D7D7);
}</string>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
